app/views/groups/members/_access_level.html.erb

```diff
@@ -0,0 +1,17 @@
+<% invalid_access_level = member.errors.include?(:access_level) %>
+<div class="form-field <%= 'invalid' if invalid_access_level %>">
```

Comment:
Same comment I sent you in Teams, it would be great if we could disable access levels that the user cannot be set to, it is better UX then allowing them to attempt to change and show and error.

Reply:
Yup I can take a look at how I'm going to do that. I will probably want to do that in a separate pr